frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_day.dart

```diff
@@ -177,7 +178,7 @@ class NewEventButton extends StatelessWidget {
           child: FlowyIconButton(
             onPressed: onCreate,
             iconPadding: EdgeInsets.zero,
-            icon: const FlowySvg(name: "home/add"),
+            icon: const FlowySvg(FlowySvgs.add_home),
```

Comment:
Shouldn’t this just be `FlowySvgs.add`. In our icon set we don’t differentiate svgs with where they are used.

Reply:
I wish that it could be `FlowySvgs.add`, but there are multiple instances of the add emoji:
![image](https://github.com/AppFlowy-IO/AppFlowy/assets/44445638/7bfd7416-0777-413d-b1e0-29ec6955e37f)

I suspected that they were different for a good reason, so I decided to append the location to the end of the name. 

I think that the right solution here is to delete one of the "add" svgs and not reduce them to one svg, but I could be wrong.

Reply:
They're duplicates, I was actually working on removing all duplicates.

Do you mind if I add a commit to your branch?

Reply:
Sure thing 💪